### PR TITLE
consolidate docs from tools repo

### PR DESCRIPTION
**Pull request by zramsay**
- moves in the README of each tool from the tools repo into the docs, creating a "Tendermint Tools" section on read-the-docs
- closes https://github.com/tendermint/tools/issues/46
- links to https://github.com/tendermint/tools/pull/51
- from the fifth commit onwards, modifications were made to the content, to go from this:
screenshot from 2017-09-13 10-01-59

to this:
screenshot from 2017-09-13 16-51-21

### CODE REVIEW

**Comment by ebuchman** [comment]
I was about to merge this but it dawned on me - can/should we not keep this in the `tools` repo ? Can we have a second readthedocs site for it? or some combine docs from multiple repos into one nice readthedocs site ?

**Comment by zramsay** [comment]
we already have 3 rtd sites so not in favour of another. The readme in root of `tools` repo can link to the rtd tools section.

**Comment by ebuchman** [comment]
I'm concerned about updates. I don't like that updating how a tool works in another repo means we have to merge to master here.  Is it possible to have the tendermint readthedocs read from two repos ?

**Comment by Greg-Szabo** [comment]
If you're concerned about updates, then shouldn't we have a `docs` repository that contains all docs for all projects? In that case, docs would be updated whenever any of the projects are updated - but it would still not affect any project.

**Comment by zramsay** [comment]
i went with something like https://github.com/tendermint/tendermint/pull/654/files#diff-85987f48f1258d9ee486e3191495582dR185 which does the trick. Note that the PR in tools is now also updated

**Comment by zramsay** [comment]
unblocked by tendermint/tools#51 being merged

File: docs/abci-cli.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
-Using the abci-cli
-==================
+Using ABCI
```
Comment (ebuchman) [inline docs/abci-cli.rst]: This really should be the ABCI-CLI ... using ABCI is something an app dev does. Here we're talking about using the CLI

**Comment by zramsay** [comment]
fixed

**Pull request merged**